vsctl/command/global.go

```diff
@@ -184,6 +191,10 @@ func mustGetEventbusID(namespace, name string) vanus.ID {
 }
 
 func mustGetNamespaceID(namespace string) vanus.ID {
+	if namespace == "" {
+		namespace = "default"
```

Comment:
“default” should be defined as a constant

Reply:
delete it,all call the function namespace is not empty